openapi.yaml

```diff
@@ -236,30 +236,11 @@ components:
       type: string
       format: email
       example: museum-lover@example.com
-    Phone:
-      description: Phone number for the ticket purchaser (optional).
-      type: string
-      example: '+1(234)-567-8910'
-    BuyMuseumTicketsRequest:
-      description: Request payload used for purchasing museum tickets.
-      type: object
-      properties:
-        ticketType:
-          $ref: "#/components/schemas/TicketType"
-        eventId:
-          description: Unique identifier for a special event. Required if purchasing tickets for the museum's special events.
-          $ref: "#/components/schemas/EventId"
-        ticketDate:
-          description: Date that the ticket is valid for.
-          $ref: "#/components/schemas/Date"
-        email:
-          $ref: "#/components/schemas/Email"
-        phone:
-          $ref: "#/components/schemas/Phone"
-      required:
-        - ticketType
-        - ticketDate
-        - email
+    BuyMuseumTickets:
```

Comment:
I think it's a bit misleading as you can buy only one ticket, not multiple of them - so it should be `BuyMuseumTicket`
Summary, description, operationId of POST operation could be updated too.

Reply:
It's a nice suggestion but let's do this as a followup - I'm breaking enough things as it is!